ssm/variational.py

```diff
@@ -412,8 +412,8 @@ def _discrete_entropy(self):
         for prms, (Ez, Ezzp1, normalizer) in \
                 zip(self.discrete_state_params, discrete_expectations):
 
-            log_pi0 = np.log(prms["pi0"] + 1e-16) - logsumexp(prms["pi0"])
-            log_Ps = np.log(prms["Ps"] + 1e-16) - logsumexp(prms["Ps"], axis=1, keepdims=True)
+            log_pi0 = np.log(prms["pi0"] + 1e-16) 
```

Comment:
To be clear, it's safe to assume these are normalized when they are set in https://github.com/lindermanlab/ssm/blob/master/ssm/lds.py#L405-L412. If we wanted to be very explicit, we could divide by the row sum here.